docs/man/containerd-config.toml.5.md

```diff
@@ -87,7 +104,7 @@ The following plugins are enabled by default and their settings are shown below.
 Plugins that are not enabled by default will provide their own configuration values
 documentation.
 
-- **[plugins."io.containerd.monitor.v1.cgroups"]** has one option __no_prometheus__ (Default: **false**)
+- **[plugins."io.containerd.monitor.task.v1.cgroups"]** has one option __no_prometheus__ (Default: **false**)
```

Comment:
Comparing 1.0 and 2.0 using vscode compare
<img width="1083" alt="Screenshot 2024-11-01 at 8 36 41 AM" src="https://github.com/user-attachments/assets/7f88928b-dd6c-4d83-a6ba-1c37b2254b35">

Reply:
These were the differences in plugins from 2.0 and 1.0 I found when doing vs-code compare for the following command:

> `contrainerd config default`

**Exist in `docs/config.md` example for cri plugins ** 
```
[plugins."io.containerd.cri.v1.runtime".containerd.runtimes.runc] ✅
[plugins."io.containerd.cri.v1.images".image_decryption] ✅
[plugins."io.containerd.cri.v1.images".pinned_images] ✅
[plugins."io.containerd.cri.v1.images".registry] ✅
[plugins."io.containerd.cri.v1.runtime".cni] ✅
[plugins."io.containerd.cri.v1.runtime".containerd] ✅
[plugins."io.containerd.cri.v1.images"] ✅
[plugins."io.containerd.cri.v1.runtime"] ✅

```
**Could not be found in the doc/man/containerd-config.toml.5.md plugins** 
```
[plugins."io.containerd.image-verifier.v1.bindir"] ❌
[plugins."io.containerd.monitor.container.v1.restart"] ❌
[plugins."io.containerd.monitor.task.v1.cgroups"] ✅ - found, but name was changed, added "task"
[plugins."io.containerd.shim.v1.manager"] ❌
```